src/arrayMethodReduce.js

```diff
@@ -5,8 +5,17 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prev = initialValue;
```

Comment:
```suggestion
    let prev = initialValue;
    
    if (arguments.length === 1) {
      prev = this[0]
    }
```
це надійніше буде і в циклі не треба зайвої метушні